app/src/main/java/org/openobservatory/ooniprobe/model/database/Measurement.java

```diff
@@ -167,4 +185,22 @@ public void setTestKeys(TestKeys testKeys) {
 		test_keys = new Gson().toJson(testKeys);
 	}
 
+	public void deleteUploadedJsons(Application a){
+		List<Measurement> measurements = Measurement.selectMeasurementsWithJson(a);
```

Comment:
What does `selectMeasurementWithJson` do?

Reply:
Select measurements that has a report file on disk